internal/pkg_npm/pkg_npm.bzl

```diff
@@ -71,9 +71,18 @@ You can pass arguments to npm by escaping them from Bazel using a double-hyphen,
 
 `bazel run my_package.publish -- --tag=next`
 
-It is also possible to use the resulting tar file file from the `.pack` as an action input via the `.tar` label:
+It is also possible to use the resulting tar file file from the `.pack` as an action input via the `.tar` label.
+To make use of this label, the `tgz` attribute must be set, and the generating `pkg_npm` rule must have a valid `package.json` file
+as part of its sources:
 
 ```python
+pkg_npm(
+    name = "my_package",
+    srcs = ["package.json"],
+    deps = [":my_typescript_lib"],
+    tgz = "my_package_tgz",
```

Comment:
I think I'd expect this to be `my_package.tgz` instead and not add the .tgz extension later? not sure if there's a reason it needs to be this way